charts/service-binding-operator/templates/tests/test-service-binding-operator-connection.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-test"
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: "{{ .Release.Name }}-test"
+      image: "quay.io/kmamgain/sboat:latest"
```

Comment:
Please include the `Dockerfile` for this image

Reply:
The Dockerfile and shell script cannot be kept in the charts directory . Can u suggest some other directory to place those.